hail_search/web_app.py

```diff
@@ -29,7 +29,7 @@ async def status(request: web.Request) -> web.Response:
     return web.json_response({'success': True})
 
 
-def init_web_app():
+async def init_web_app():
```

Comment:
I didn't change this yet, but `load_globals()` uses `hail` functions so we should add `hl.init()` to this function.

Reply:
I think that messes up unit tests, so I would be careful and do it as its own PR

Reply:
I added `hl.init(idempotent=True)`, which I think is equivalent to what's happening under the hood now.

Reply:
Tests are ok!